plugin/cancel/cancel.py

```diff
@@ -45,7 +45,8 @@ def pre_phrase(phrase: Phrase):
 
     # Check if the phrase is before the threshold
     if ts_threshold != 0:
-        start = getattr(words[0], "start", phrase["_ts"])
+        # NB: mimic() and Dragon don't have this key.
+        start = getattr(words[0], "start", None) or getattr(phrase, "_ts", ts_threshold)
```

Comment:
This behaves very differently to previous. This is now looking for `phrase._ts` instead of `phrase["_ts"]`.

Does a Phrase have both key access and attribute access?

Reply:
You're totally right. This should have been a `.get` on phrase. Good catch.
https://github.com/talonhub/community/pull/1633